SG Policy.md

```diff
@@ -40,13 +40,16 @@ G. As an exception, [Steering Group] decisions on confidential matters which hav
 
 A. The [Steering Group] may define policies and agreements that govern the operation of the WHATWG. Policies must not conflict with the [Steering Group Agreement]. If they do, the [Steering Group Agreement] takes precedence.
 
-B. Other than the [Steering Group Agreement] and the [WHATWG Principles], the [Steering Group] may adopt, amend, or revoke Policies or Agreements by [consensus]. The same applies to this Steering Group Policy.
+B. Other than the [Steering Group Agreement] and the [WHATWG Principles], the [Steering Group] may adopt, amend, or revoke policies or agreements by [consensus]. The same applies to this Steering Group Policy.
 
 C. The canonical copies of all WHATWG policies and agreements must be kept in the [Steering Group]'s repository. Policies may also be published elsewhere, such as on <https://whatwg.org/>.
 
-D. Changes to policies and agreements should be made by pull request associated with an issue. Changes must be reviewed by a [Steering Group Representative] and may only be merged by [Consensus] or [unanimous consent] of the [Steering Group], as set forth in the [Steering Group Agreement].
+D. Changes to policies and agreements should be made by pull request associated with an issue that has a title prefix of "Policy Change: ". Changes must be reviewed by a [Steering Group Representative] and may only be merged when:
```

Comment:
Should we clarify that both conditions (if the second applies) must be met? I wanted to suggest "when all of the following conditions are true" in the style of spec prose, but it doesn't read well with a conditional condition...

Reply:
We could make it a numbered list?

Reply:
Simple enough, yes!